### DISCUSSION TITLE
Power Platform Import Solution task in ADO pipeline failing with error  The remote server returned an error: (413) Request Entity Too Large.

### DISCUSSION THREAD

**Discussion by pkpaul555** [microsoft/powerplatform-build-tools#907] [Q&A]
Hi All,

We are trying to deploy solution file to Dynamics 365 using ADO pipeline. The 'Power Platform Import Solution' task in ADO pipeline is failing with error  'The remote server returned an error: (413) Request Entity Too Large'. 
The solution file size 127 MB. Please let me know how to address this issue.

**Comment by petrochuk** [comment]
It looks like the issue is very specific to your solution so I would open Support Case with Microsoft who can look into this.

**Reply by pkpaul555** [reply]
Hi petrochuk, can you please provide a link where I can raise a support case with Microsoft

**Reply by petrochuk** [reply]
> Hi petrochuk, can you please provide a link where I can raise a support case with Microsoft

[Get Help + Support](https://learn.microsoft.com/en-us/power-platform/admin/get-help-support)
